README.md

```diff
@@ -0,0 +1,85 @@
+| Diego Petrucci          | [@diegopetrucci](https://twitter.com/diegopetrucci)           |
+| Ilya Puchka             | [@ilyapuchka](https://twitter.com/ilyapuchka)                 |
+| Jason Dobo              |                                                               |
+| Martin Nygren           |                                                               |
+| Michael Brown           | [@mluisbrown](https://twitter.com/mluisbrown)                 |
+| Rui Peres               | [@peres](https://twitter.com/peres)                           |
+| Sergey Shulga           | [@SergDort](https://twitter.com/SergDort)                     |
+| Viorel Mihalache        | [@viorelMO](https://twitter.com/viorelMO)                     |
+| Wagner Truppel          |                                                               |    
+| Witold Skibniewski      | [@mr_witold](https://twitter.com/mr_witold)                   |
+
+
+## 2. Which squad each individual belongs to and availability
+
+By definition members work on their respective squad, although they are free to work in different squads if the work load justifies it.
+
+
+| Squad Name                    | Members                  | Availability |  
+|-------------------------------|--------------------------| ------------ |
+| SDK                           | David, Viorel, Martin    |    3/4       |
+| Consultation Experience       | Ilya                     |    1/1       |
+| Healthcheck                   | Ben, Wagner              |    2/2       |
+| Core                          | Witold, Jason            |    2/4       |
+| Professional Services         | Danilo                   |    1/1       |
+| GP at Hand                    | Diego                    |    1/1       |
+| Project Avalon                | Sergey                   |    1/1       |
+| Project Minkowski             | Anders                   |    1/2       |
+| Project Bayes                 | Michael                  |    1/1       |
+
+## 3. OSS maitained projects
+
+| Project name                  | Owner(s)                 | Stars        |  
+|-------------------------------|--------------------------| ------------ |
+| Bento                         | Anders, David, Serg      | [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/Bento.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/Bento/stargazers/)             |
+| DrawerKit                     | David, Wagner            |    [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/DrawerKit.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/DrawerKit/stargazers/)       |
+| ReactiveFeedback              | Anders, Serg             |    [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/ReactiveFeedback.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/ReactiveFeedback/stargazers/)        |
+
+## 4. OSS maintained projects
```

Comment:
This is a repeat of section 3, so it should be removed. And the following section numbers adjusted accordingly.